en/misc.ftl

```diff
@@ -267,6 +267,20 @@ first-time-user-email-hero-primary-text = As a { -brand-name-firefox } user, you
 first-time-user-email-hero-secondary-text = Manage all your masks from your { -brand-name-relay } dashboard.
 first-time-user-email-hero-cta = View your dashboard
 
+## Email sent to people onboarding that receive their first forwarded email.
+
+our-promise-header = Our promise to you
+our-promise-content = { -brand-name-relay } email masks forward to your email inbox any messages sent to that email mask—without revealing your real email address to the sender.
+what-can-you-do-with-relay-title = What can you do with { -brand-name-firefox-relay }?
+what-can-you-do-with-relay-point-one = Get emails without revealing your real email address to senders
+what-can-you-do-with-relay-point-one-subpoint-html = Just share an email mask instead. Email masks will forward emails to your inbox. You can create masks directly on { -brand-name-firefox }, with the <a href="{ $url }" { $attrs }>{ -brand-name-relay } extension</a>, or on your { -brand-name-relay } dashboard.
```

Comment:
Usually we include a comment where the URL points to, to give localizer some context.